### ISSUE TITLE
Label text is now falling out of the flex container borders:

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud-libraries/nextcloud-vue#4896]
> Label text is now falling out of the flex container borders:
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/93392545/381ca21d-5971-416e-8302-2239bc2d99ca)
With flex-shrink reverted for `checkbox-content__text` class:
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/93392545/158cffd8-ff72-4c4a-ae2d-cbcbc7673d77)

_Originally posted by @Antreesy in https://github.com/nextcloud-libraries/nextcloud-vue/issues/4760#issuecomment-1833690743_

**Comment by Antreesy** [comment]
Should be fixed already by https://github.com/nextcloud-libraries/nextcloud-vue/pull/4862
